src/components/plots/gridPlot.ts

```diff
@@ -96,6 +96,54 @@ export module Plots {
     protected _generateDrawSteps(): Drawers.DrawStep[] {
       return [{attrToProjector: this._generateAttrToProjector(), animator: this._getAnimator("cells")}];
     }
+
+    public x(): Plots.AccessorScaleBinding<any, number>;
+    public x(x: number | _Accessor): Grid;
+    public x(x: any | _Accessor, xScale: Scale<any, number>): Grid;
+    public x(x?: number | _Accessor | any, xScale?: Scale<any, number>): any {
+      if (x == null) {
+        return super.x();
+      }
+      super.x(<any> x, xScale);
+      if (xScale instanceof Scales.Category) {
```

Comment:
This will get fixed when take the weirdness out of `Drawer`s, but in the meantime :(